examples/DotNetCore/WebDemoWithEventHub/WebDemoWithEventHub/Startup.cs

```diff
@@ -22,6 +22,7 @@ public void ConfigureServices(IServiceCollection services)
 
             // We add a Settings model to the service container, which takes its values from the applications configuration.
             services.Configure<Settings>(Configuration.GetSection("WebDemo:Settings"));
+            services.AddSingleton<EventHubService>();
```

Comment:
Please add a comment to explain this call

Reply:
Added comment.

---
In reply to: [462050387](https://github.com/Azure/AppConfiguration/pull/329#discussion_r462050387) [](ancestors = 462050387)